packages/eas-cli/src/graphql/queries/UserQuery.ts

```diff
@@ -28,6 +28,10 @@ export const UserQuery = {
                 }
                 ... on SSOUser {
                   username
+                  primaryAccount {
```

Comment:
can just change the `... on User` to `... on UserActor` and remove this SSOUser spread

Reply:
We may need to add SSOUser to `additionalTypenames` below as well.

Reply:
Done